staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -419,7 +457,7 @@ var (
 	EnvelopeKMSv2ServiceFactory = envelopekmsv2.NewGRPCService
 )
 
-func kmsPrefixTransformer(config *apiserverconfig.KMSConfiguration, stopCh <-chan struct{}) (value.PrefixTransformer, healthChecker, error) {
+func kmsPrefixTransformer(config *apiserverconfig.KMSConfiguration, stopCh <-chan struct{}) (value.PrefixTransformer, healthChecker, bool, bool, error) {
```

Comment:
super nit, but name the return values or godoc it or something (and maybe order them as kmsv1, kmsv2?)

Reply:
same comments up the call chain, sorry... two undocumented bools in a surprising order is asking for accidents

Reply:
alternatively, make a struct to hold info about the loaded encryption config with KMSV1 and KMSV2 bools, it looks like https://github.com/kubernetes/kubernetes/pull/112050 is going to add ~3 more return values that need plumbing (max kms timeout, content hash, and totalKMSPlugins (though I'm not sure why we need the total kms count)